MAUI/DataGrid/Column-types.md

```diff
@@ -0,0 +1,916 @@
+
+### Binding options
+
+Display content of the DataGridColumn is determined from the `DataGridColumn.DisplayBinding` property. It gets or sets display binding that associates the DataGridColumn with a property in the data source. 
+
+The actual bound value of the GridColumn is determined from `DataGridColumn.ValueBinding` property. It gets or sets value binding that associates the DataGridColumn with a property in the data source.  
+
+#### Mapping column to particular property
+
+The `DataGridColumn.MappingName`associates the DataGridColumn with a property available in the underlying data source. While setting MappingName alone to the SfDataGrid, the `DataGridColumn.DisplayBinding` will be automatically generated based on the MappingName. Data manipulation operations like sorting, filtering, and grouping will be done based on the MappingName property.
+
+To format cell content, use the converter of the `DataGridColumn.DisplayBinding` to customize the cell content. The following code example appends the text “Customer” along with the Customer ID:
+
+{% highlight xaml %}
+<ContentPage.Resources>
+    <ResourceDictionary>
+        <local:DisplayBindingConverter x:Key="displayBindingConverter" />
+    </ResourceDictionary>
+</ContentPage.Resources> 
+
+<syncfusion:SfDataGrid x:Name="dataGrid"
+                       ItemsSource="{Binding OrdersInfo}">
+
+    <syncfusion:SfDataGrid.Columns>
+        <syncfusion:DataGridTextColumn MappingName="CustomerID" 
+                                   DisplayBinding="{Binding CustomerID, 
+                                   Converter={StaticResource displayBindingConverter}}" />
+    </syncfusion:SfDataGrid.Columns>
+</syncfusion:SfDataGrid> 
+{% endhighlight %}
+
+{% highlight c# %}
+public class DisplayBindingConverter : IValueConverter
+{
+    public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        if (value != null)
+            return "Customer:" + value.ToString();
+        return null;
+    }
+
+    public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        return value.ToString().Substring(9);
+    }
+}
+{% endhighlight %}
+
+### Header customizations
+
+#### HeaderText
+
+To customize the display content of the header cell, use the `DataGridColumn.HeaderText` property. It specifies the text displayed in the column header. If header text is not defined, then `DataGridColumn.MappingName` will be assigned to the header text and will be displayed as column header.
+
+#### HeaderTextAlignment
+
+To get or set the TextAlignment of the header cell, use the `DataGridColumn.HeaderTextAlignment` property. The default alignment is `Center`. It can be customized as `Start` or `End`.
+
+#### HeaderTemplate
+
+Based on the requirement, the header cell can be customized using the `DataGridColumn.HeaderTemplate` property. To customize header cell by loading a template in the header cell, follow the code example:
```

Comment:
Based on the requirement, the header cell can be customized using the `DataGridColumn.HeaderTemplate` property. To customize the header cell by loading a template in the header cell, follow the code example: